gtp2ogs.js

```diff
@@ -305,6 +344,16 @@ class Game {
             this.state = gamedata;
             check_for_move();
         });
+        this.socket.on('game/' + game_id + '/clock', (clock) => {
```

Comment:
Perhaps leave it where it is , but don't run loadClock(), simply update this.state.clock so the next loadState( ) has the info handy?